src/components/Segmented/Segmented.module.css

```diff
@@ -0,0 +1,49 @@
+.segmented {
+  position: relative;
+  display: flex;
+  align-items: stretch;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  user-select: none;
+  width: 100%;
+  margin: 0;
+  gap: 2px; /* does not change with theme */
+  padding: 2px; /* does not change with theme */
+  border-radius: calc(var(--shape-border-radius-md, 4) * 1px);
+  background-color: color-mix(in srgb, var(--palette-common-grey-300, #e6e6e6), transparent 50%);
+}
+
+.segmentedOption {
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  gap: calc(var(--spacing-gap-xs, 4) * 1px);
+  padding: calc(var(--spacing-padding-sm, 8) * 1px);
+  min-width: calc(var(--shape-size-5xl, 64) * 1px);
+  border-radius: calc(var(--shape-border-radius-md, 4) * 1px);
+  cursor: pointer;
+  transition: background-color 0.25s ease-in-out;
+}
+
+.segmentedOption:hover {
+  background-color: color-mix(in srgb, var(--palette-common-grey-600, #aaaaaa), transparent 75%);
+}
+
+.segmentedOption.selected {
```

Comment:
Why do we use a double class here? it should just be enough ".selected" and ".vertical"

Reply:
I can only put selected, if I include also vertical the whole shape of the control goes vertical, which is not the expected behaviour.

Reply:
I tried but in both cases, the style did not get the expected behavior. 

For vertical, the whole shape of the control aligns vertically.
For selected, the background color collides when on hover, keeping the hover color rather than the selected color